### ISSUE TITLE
When is release v0.9.7 expected?

### ISSUE THREAD

**Issue by easwars** [envoyproxy/go-control-plane#360]
At gRPC-Go, we are working on using certain v3 xDS APIs which are not part of the v0.9.6 release, but are part of the v0.9.7 release. I'd like to know when v0.9.7 is planned for?

Thank you.

**Comment by ashutosh-narkar** [comment]
We are looking to use [this](https://github.com/envoyproxy/envoy/pull/13131) new feature to send request body as raw bytes to the external authorization server. It would be great if the new release could include that as well. Thanks !

**Comment by kyessenov** [comment]
Sure, I'll tag a release once #361 is merged.
